src/pe_source/data/helpers/redact_pii.py

```diff
@@ -0,0 +1,560 @@
+"""Functions to redact PII from a dataframe."""
+
+# Standard Python Libraries
+import re
+
+# Third-Party Libraries
+from presidio_analyzer import AnalyzerEngine
+from presidio_anonymizer import AnonymizerEngine
+import scrubadub
+import scrubadub.detectors.date_of_birth
+
+# List of regexes to identify each state's Drivers License format in a larger string
+CA = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{7}(?=$|\s)"]
+CO = [r"(?:(?<=\s)|(?<=^))\d{2}-\d{3}-\d{4}(?=$|\s)"]
+FL = [
+    r"(?:(?<=\s)|(?<=^))[a-zA-Z] \d{3} \d{3} \d{3} \d{3}(?=$|\s)",
+    r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{3}-\d{3}-\d{2}-\d{3}-\d(?=$|\s)",
+    r"(?:(?<=\s)|(?<=^))[a-zA-Z]-\d{3}-\d{3}-\d{3}-\d{3}(?=$|\s)",
+]
+HI_NE_VA = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{8}(?=$|\s)"]
+IA = [r"(?:(?<=\s)|(?<=^))\d{3}[a-zA-Z]{2}\d{4}(?=$|\s)"]
+ID = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]{2}\d{6}[a-zA-Z](?=$|\s)"]
+IL = [
+    r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{3}-\d{4}-\d{4}(?=$|\s)",
+    r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{11}(?=$|\s)",
+]
+IN = [r"(?:(?<=\s)|(?<=^))\d{4}-\d{2}-\d{4}(?=$|\s)"]
+KS = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{2}-\d{2}-\d{4}(?=$|\s)"]
+KY = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{2}-\d{3}-\d{3}(?=$|\s)"]
+MD = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]-\d{3}-\d{3}-\d{3}-\d{3}(?=$|\s)"]
+MI = [r"(?:(?<=\s)|(?<=^))[a-zA-Z] \d{3} \d{3} \d{3} \d{3}(?=$|\s)"]
+MN_FL_MD_MI = [r"(?:(?<=\s)|(?<=^))[a-zA-Z]\d{12}(?=$|\s)"]
```

Comment:
Since FL, MD, and MI already have their own regexes, I think it's cleaner to include this regex in those individual state regexes and rename this one from `MN_FL_MD_MI` to just `MN`.  I suppose that's a matter of preference, so feel free to disagree with me here.

Reply:
@dav3r issue resolved in the following commit: [2ccd8e8](https://github.com/cisagov/pe-reports/pull/190/commits/2ccd8e832633b9d735dabf21daff9be9299da256)

Reply:
That looks great! 👍